sequence_task_3/student/include/impl.h

```diff
@@ -8,45 +8,120 @@ class DynamicPlaylist
 {
 public:
     /** @todo Member traits */
-
+    using size_type = typename Container::size_type;
+    using value_type = typename Container::value_type;
+    using reference = typename Container::reference;
+    using const_reference = typename Container::const_reference;
+    using iterator = typename Container::iterator;
+    using const_iterator = typename Container::const_iterator;
+    using difference_type = typename Container::difference_type;
     /** @todo Iterators */
+    iterator begin()
+    {
+        return m_tracklist.begin();
+    };
+    iterator end()
+    {
+        return m_tracklist.end();
+    };
+    const_iterator begin() const
+    {
+        return m_tracklist.cbegin();
+    };
+    const_iterator end() const
+    {
+        return m_tracklist.cend();
+    };
 
     DynamicPlaylist() = default;
 
     /** @todo Constructor from two iterators*/
-
+    template<class T>
+    DynamicPlaylist(T first, T second)
```

Comment:
1. Use initialization list
2. Use sequence constructor

Reply:
More details please. I don't understand what is wrong .

Reply:
You may use the constructor directly and initialize container with two iterators (check "Containers common/Constructors" section of the training)
For that you may use member initializer list (https://www.learncpp.com/cpp-tutorial/constructor-member-initializer-lists/)